_tools/gulp/helpers/args.js

```diff
@@ -43,6 +43,13 @@ if (format === '' && !args.silent) {
   console.log('If processing for a specific format, use the --format argument, e.g. gulp --format printpdf. No hyphens.')
 }
 
+// Get whether we are using merged.html
+let merged = true
+if (args.merged && args.merged.trim() !== '') {
+  merged = args.merged
```

Comment:
I haven't tested here @arthurattwell , but does this ever cause a TypeError, since `merged` is a boolean here on line 47, will be cast to a string on line 49, and then we're comparing it to a string on line 23 of pdf.js

Reply:
@LouiseSteward Great catch. I think I was lucky this was working. I've fixed this to initialise the var as a string in https://github.com/electricbookworks/electric-book/pull/678/commits/9e7f885972631724e1bfef95be44dba7a0c34290. It does feel a little weird to use `true`/`false` as strings, but that's simpler than trying to keep this a Boolean, and `true`/`false` as command-line options seem appropriate.